### ISSUE TITLE
Four false-negative modes when running the CLEANUP.md checks on a Dockerised store

### ISSUE THREAD

**Issue by splisko** [disrex-group/stylesmuggler-mitigation#1]
Thanks for publishing this — the CLEANUP.md ordering (preserve before contain, clean before mitigate) is the part that most write-ups get wrong, and it was genuinely useful during an incident.

Reporting four **false-negative modes** I hit running these checks against a Dockerised Magento 2.4.8 deployment. None of these involves attacker-usable detail, so per SECURITY.md I'm filing publicly. I have emailed support@disrex.nl separately about two detection-evasion items.

The common thread: on a containerised store several checks *cannot run*, and there is nothing in the documented commands that distinguishes "ran and found nothing" from "did not run". Both look like silence, and silence reads as clean.

### 1. `ps` is absent from slim PHP images, and its absence is silent

`CLEANUP.md` §1 and the IOC.md variant both shell out to `ps`. Official `php:*-fpm` images and most slim bases don't ship procps, so the pipeline exits non-zero, prints nothing, and looks identical to a clean result.

Suggested wording: note that container processes are visible in the **host** PID namespace, so the reliable place to run §1 is the host, walking `/proc` directly rather than calling `ps`. That also removes the need to install anything on a compromised box.

### 2. `$MAGENTO_ROOT/var/log` and `var/report` are frequently bind mounts

§1 and §4c both grep `"$MAGENTO_ROOT/var/report/"` and `"$MAGENTO_ROOT/var/log/"`. On a typical Docker deployment those are bind-mounted from elsewhere on the host, so from the host's point of view `$MAGENTO_ROOT/var/log` may not exist at all — `grep -rl` on a missing directory prints nothing and returns 1.

I had a genuinely poisoned `system.log` sitting one directory away from where the documented command looked.

Suggested wording: add a note to derive the real paths from `docker inspect <container> --format '{{range .Mounts}}{{.Source}} {{.Destination}}{{end}}'`, or to run §4c *inside* the container where the paths are canonical.

### 3. §4c's media check reports presence, not reachability

```bash
find "$MAGENTO_ROOT/pub/media" -name '*.ph*' -type f
```

This finds files but doesn't answer the two questions that decide urgency: is the file reachable over HTTP, and does it execute? Those have very different answers in practice — a file can be blocked (404), served as readable source (200 returning the PHP as text), or executing.

That distinction mattered a lot to me: it separated "inert, clean up whenever" from "this is serving the dropper's source to anyone who asks".

Suggested addition — a follow-up probe per hit:

```bash
curl -s -o /tmp/p -w '%{http_code}\n' "https://YOURSTORE/media/<relative-path>"
grep -qa '<?php\|<?=' /tmp/p && echo "SOURCE DISCLOSED"
```

Also worth noting that `-name '*.ph*'` won't match `.inc` or `.phar`.

### 4. Suggested framing change: make "could not run" a distinct outcome

Everything above collapses into one documentation-level point. §1 currently ends with:

> **No hits anywhere?** You are probably fine. Apply the snippets and move on.

That conclusion is only sound if the commands actually executed. On a containerised store, several plausibly didn't. Given how load-bearing that sentence is, it might be worth an explicit caveat — something like *"if any command above errored, produced no output because a path or binary was missing, or you could not read other users' `/proc`, treat the result as unknown rather than clean, and re-run it where it can actually execute."*

Happy to open a PR for any of the above if useful. I have a rewritten checker that implements all four (`[SKIP]` as a first-class outcome, distinct from pass, forcing a non-zero exit) and I'm glad to contribute it or just the doc patches, whichever you'd prefer.
